core/src/main/java/io/grpc/CallOptions.java

```diff
@@ -333,6 +371,7 @@ public String toString() {
     toStringHelper.add("executor", executor != null ? executor.getClass() : null);
     toStringHelper.add("compressorName", compressorName);
     toStringHelper.add("customOptions", Arrays.toString(customOptions));
+    toStringHelper.add("isWaitForReady", isWaitForReady());
 
```

Comment:
I will change it to 
    `toStringHelper.add("waitForReady", isWaitForReady());`
to follow the convention.